person.rb

```diff
@@ -7,15 +9,22 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @nameable = Nameable.new
```

Comment:
* Kindly add all decoration design patterns as stated  in the [project requirement ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Interface,trim%20the%20word.)
* Also test your decorators with the code snippet below 👇 
![image](https://user-images.githubusercontent.com/52497246/220451079-bd05235e-6799-4920-a277-e7468afaf434.png)